cmd/synthetic-monitoring-agent/main.go

```diff
@@ -244,26 +244,24 @@ func run(args []string, stdout io.Writer) error {
 		return checksUpdater.Run(ctx)
 	})
 
-	if features.IsSet(feature.AdHoc) {
-		adhocHandler, err := adhoc.NewHandler(adhoc.HandlerOpts{
-			Conn:           conn,
-			Logger:         zl.With().Str("subsystem", "adhoc").Logger(),
-			Backoff:        newConnectionBackoff(),
-			Publisher:      publisher,
-			TenantCh:       tenantCh,
-			PromRegisterer: promRegisterer,
-			Features:       features,
-			K6Runner:       k6Runner,
-		})
-		if err != nil {
-			return fmt.Errorf("Cannot create ad-hoc checks handler: %w", err)
-		}
-
-		g.Go(func() error {
-			return adhocHandler.Run(ctx)
-		})
+	adhocHandler, err := adhoc.NewHandler(adhoc.HandlerOpts{
```

Comment:
Should we print a simple warning saying it is now deprecated and has no effect? It can help as a reminder for operators to garbage-collect the flag.

Reply:
good point, thanks